sixtyfps_runtime/rendering_backends/gl/glwindow.rs

```diff
@@ -42,6 +42,8 @@ pub struct GLWindow {
     // This cache only contains textures. The cache for decoded CPU side images is in crate::IMAGE_CACHE.
     pub(crate) texture_cache: RefCell<TextureCache>,
 
+    fps_counter: Option<Rc<FPSCounter>>,
```

Comment:
does it really belong in the window?  or should it be in the backend?

Reply:
In principle the data is per-window - different windows might render at different frame rates.

Eventually I'd also rather print this as overlay text instead of going via stdout.

Without this patch, try this on your Linux system:

  * set the `GALLIUM_HUD=fps` environment variable
  * Run any SixtyFPS app with `SIXTYFPS_BACKEND=GL`
  * Watch the fps counter and graph :-)

I think this would be nice to have (even without the graph, just text) and then it's per-window. That could be for example `SIXTYFPS_DEBUG_PERFORMANCE=overlay`.